src/main/java/pl/betoncraft/betonquest/conversation/Conversation.java

```diff
@@ -469,7 +469,7 @@ public void run() {
 			// would leave it active while the conversation is not
 			// started, causing it to display "null" all the time
 			try {
-				String name = plugin.getConfig().getString("default_conversation_IO");
+				String name = data.getConversationIO();
 				Class<? extends ConversationIO> c = plugin.getConvIO(name);
 				if (c == null) {
```

Comment:
This check is no longer needed, validity of selected `ConversationIO` is checked at loading-time now, right?